src/components/Item/styles.js

```diff
@@ -0,0 +1,73 @@
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import Icon from '../Icon';
+
+const styles = {
+  color({ size }) {
+    if (size === 'mini') return 'var(--f-color-text--secondary)';
+
+    return 'var(--f-color-text--primary)';
+  },
+  xPadding({ size }) {
```

Comment:
why xPadding? just curious about the naming :)

Reply:
its used for top/bottom so maybe `yPadding` 😄

Reply:
ops. It should be `yPadding` indeed. Updated.

y axis = horizontal. Is it too explicit?